.devcontainer/Dockerfile

```diff
@@ -47,9 +47,9 @@ RUN export PORTER_HOME=/home/vscode/.porter \
 ENV PATH /home/vscode/.porter/:$PATH
 
 # Install requirements
-COPY ["requirements.txt", "requirements-dev.txt", "/tmp/pip-tmp/" ]
```

Comment:
not sure in github how to comment on the move so I'm doing it here... 

I'm a bit confused here... the requirements-dev.txt is supposed to be a more generic python requirements file -- containing things like pre-commit etc. it is not api specific, so it should be in the root... and the root requirements one tied the api and the dev one together (eg. included the api specific and the dev general requirements).  If we have a specific one for the resource processor, the main (root one) should include links to both the dev requirements, the api requirements, and the resource processor requirements - e.g. a one stop shop for all requirements you need for development on your dev machine (installed on docker but also easily installable in a non-docker environment for those who prefer pycharm or other ides).

Also, did this now overwrite the root or the API requirements.txt?

Reply:
per discussion